apps/smbmount.sh

```diff
@@ -28,7 +28,7 @@ fi
 # functions
 add_mount() {
 # check if mounting slots are available
-if [ -n "$(grep /mnt/smbshares/1 /etc/fstab)" ] && [ -n "$(grep /mnt/smbshares/2 /etc/fstab)" ] && [ -n "$(grep /mnt/smbshares/3 /etc/fstab)" ]
+if [ grep -q /mnt/smbshares/1 /etc/fstab ] && [ grep -q /mnt/smbshares/2 /etc/fstab) ] && [ grep -q /mnt/smbshares/3 /etc/fstab ]
```

Comment:
I don't think you need the [ ]here as it's no if statment with a switch like -n -z -f or similar.

Reply:
thanks for the advice. Travis is fixed finally!